kubernetes-101/deploying-pod/step7.md

```diff
@@ -1,41 +1,53 @@
 In this section we will show you how to access the logs for a particular pod and how to tidy up and delete your deployments.
+
 # Viewing a Pod's logs
 
 On many occasions it will be useful to view the logs of the pods running.
-## Task 
+
+## Task
+
 First, check your pod is running by running `kubectl get pods`{{execute}}. To access the logs for a particular pod (e.g. http) we can run the following code `kubectl logs http`{{execute}}.
 
-You should see an initial log showing `Web Server started. Listening on 0.0.0.0:80` 
+You should see an initial log showing `Web Server started. Listening on 0.0.0.0:80`
 
 We will show more details later on how to access this pod externally.
 
 ### Exercise 1 - Accessing pod logs
+
 Try to access the logs for the **http-yaml** pod. You should see a similar output.
 
 # Deleting a pod
 
 We will now discuss how to delete pods once we have finished with resources.
 
 ## Delete a single pod
+
 First, check the list of pods by running `kubectl get pods`{{execute}}.
 
 We can then delete each of the pods in two ways.
-###Kubectl CLI
+
+### Kubectl CLI
+
 We can delete a pod one by one by running `kubectl delete pod pod-name`. For our http server we can do this by running `kubectl delete pod http`{{execute}}.
 
-###Yaml file
```

Comment:
These were the only things that required fixing. The space is needed. But consistent whitespace around headers also makes it easier to notice and prevent these from getting accidentally treated as inline text too.